Apps/Playground/playground-shared/App.tsx

```diff
@@ -127,6 +127,7 @@ const EngineScreen: FunctionComponent<ViewProps> = (props: ViewProps) => {
     }
   }, [engineViewCallbacks]);
 
+  // @refresh reset
```

Comment:
Couldn't this just be done inside EngineView.tsx?

Reply:
I tried but it doesn't work.